src/main/resources/assets/sodium/lang/ru_ru.json

```
@@ -0,0 +1,66 @@
+  "sodium.options.view_bobbing.tooltip": "Если включено, обзор игрока будет покачиваться при движении. Отключение этого параметра может помочь людям, страдающим от укачивания во время игры.",
+  "sodium.options.attack_indicator.tooltip": "Управляет расположением индикатора атаки на экране.",
+  "sodium.options.autosave_indicator.tooltip": "Если включено, при сохранении игрой мира на диск будет показываться индикатор.",
+  "sodium.options.graphics_quality.tooltip": "Качество графики по умолчанию управляет некоторыми наследующими настройками и необходимо для совместимости с модами. Опции ниже со значением \"По умолчанию\" будут использовать этот параметр.",
+  "sodium.options.clouds_quality.tooltip": "Управляет качеством отрисовки облаков в небе.",
+  "sodium.options.weather_quality.tooltip": "Управляет качеством эффектов дождя и снега.",
+  "sodium.options.leaves_quality.name": "Качество листьев",
+  "sodium.options.leaves_quality.tooltip": "Управляет качеством листьев.",
+  "sodium.options.particle_quality.name": "Качество частиц",
+  "sodium.options.particle_quality.tooltip": "Управляет количеством частиц, которые могут присутствовать на экране в любой момент времени.",
+  "sodium.options.smooth_lighting.tooltip": "Управляет мягким освещением и затенением блоков. Это немного увеличивает время сборки чанка, однако не влияет на частоту кадров.",
+  "sodium.options.biome_blend.value": "%s блок(ов)",
+  "sodium.options.biome_blend.tooltip": "Управляет диапазоном анализа биомов для расцветки блока. Более высокие значения сильно увеличивают затрачиваемое на сборку чанков время ради незначительного улучшения качества.",
+  "sodium.options.entity_distance.tooltip": "Управляет дальностью видимости сущностей. Высокие значения увеличивают расстояние отображения за счёт частоты кадров.",
+  "sodium.options.entity_shadows.tooltip": "Если включено, простые тени будут отображаться под мобами и другими сущностями.",
+  "sodium.options.vignette.name": "Виньетирование",
+  "sodium.options.vignette.tooltip": "Если включено, на экране игрока будет отображаться эффект виньетирования. Вряд ли повлияет на частоту кадров, если ваш ГП не ограничен скоростью заполнения пикселей.",
+  "sodium.options.mipmap_levels.tooltip": "Управляет количеством уровней детализации для текстур блоков. Более высокие значения обеспечивают лучший вид на расстоянии, но могут негативно повлиять на производительность с большим количеством анимированных текстур.",
+  "sodium.options.use_block_face_culling.name": "Отбраковка поверхностей блоков",
+  "sodium.options.use_block_face_culling.tooltip": "Если включено, на рендеринг будут выводиться только видимые для камеры поверхности блоков. Это может исключить большое количество поверхностей на ранней стадии отрисовки, экономя пропускную способность памяти и время для ГП. Некоторые пакеты ресурсов могут иметь проблемы с этой опцией, поэтому попробуйте отключить её, если вы видите дыры в блоках.",
```

Better consistency wth the official localization
```suggestion
  "sodium.options.use_block_face_culling.tooltip": "Если включено, на рендеринг будут выводиться только видимые для камеры поверхности блоков. Это может исключить большое количество поверхностей на ранней стадии отрисовки, экономя пропускную способность памяти и время для ГП. Некоторые наборы ресурсов могут иметь проблемы с этой опцией, поэтому попробуйте отключить её, если вы видите дыры в блоках.",
```